client/src/components/Tabs/Tabs.js

```diff
@@ -137,11 +140,17 @@ Tabs.propTypes = {
   id: React.PropTypes.string.isRequired,
   defaultActiveKey: React.PropTypes.string,
   extraClass: React.PropTypes.string,
+  data: React.PropTypes.shape({
+    hideNav: React.PropTypes.bool,
```

Comment:
I think making this its own boolean prop would be good. As discussed in Slack we should be able to add `hideNav` to the schema data defaults then use `setSchemaState(['hideNav' => true])`, since "Any passed keys that are not defined in {\@link getSchemaDataDefaults()} are ignored."